README.md

```diff
@@ -736,12 +735,12 @@ public function getRelationships($user)
 }
 ```
 
-This way, the contacts of a user will only be loaded when the given relationship's `data` key is present in the response
+This way, the contacts of a user will only be loaded when the given relationship's `data` key is present in the response,
 allowing your API to be as effective as possible.
 
 #### Injecting metadata into documents
 
-Metadata can be injected into documents on-the-fly. This might come in handy if you want to customize or decorate your
+Metadata can be injected into documents on-the-fly. This can be handy if you want to customize or decorate your
```

Comment:
I am curious why you prefer "can be handy" here? The "might come in handy" came from another Hactoberfest submission and it sounded for me very well.